src/client/src/translations/en.json

```diff
@@ -2,7 +2,6 @@
   "FunctionNameError.invalidFunctionStartLetter": "Name may only start with letters or numbers",
   "FunctionNameError.invalidRegex": "Name may only contain letters, numbers or dashes",
   "about.about": "About",
-  "about.reportAnIssue": "Give Feedback or Report an issue",
```

Comment:
Why was this one removed?

Reply:
I think this needs to be removed. When we changed the licenses from a plain list to modals, this was changed to `"about.reportAnIssueLabel": "Give Feedback"`, which is already present in the file.
I am not very sure though.
@annielovesjs Do you have any comments on this one?

Reply:
I don't think manage translations works properly so I have to manually change the text in translation files. I just added this one back

Reply:
manage translations didn't effect these files for some reason so I just added it back in